packages/api/src/name.ts

```diff
@@ -69,7 +69,10 @@ export async function nameGet (request: Request, env: Env): Promise<Response> {
     if (error instanceof HTTPError) {
       throw (error)
     } else {
-      throw new HTTPError('please try again', 500)
+      const errorString = error instanceof Object
+        ? JSON.stringify(error)
+        : 'unknown error'
+      throw new HTTPError(`Unable to retrieve name. ${errorString}`, 500)
```

Comment:
Is the more likely scenario here that `error` is an instance of `Error`, and therefore we should use `error.message`? Calling `JSON.stringify(error)` when `error` is an `Error` just gives `'{}'`.